api/comment_threads.rb

```diff
@@ -82,3 +93,15 @@
   thread.destroy
   thread.to_hash.to_json
 end
+
+get "#{APIPREFIX}/courses/*/stats"  do |course_id| # retrieve stats by course
+  begin
+    threads = Content.where(_type: "CommentThread", course_id: course_id)
+    active_threads = threads.where(:last_activity_at.gte => 1.day.ago)
+    course_stats = {
+      "num_threads" => threads.count,
```

Comment:
If this change came from upstream, let's not address the nit. The goal is to keep our fork as close to upstream as possible.